src/test/createUserMutation.test.ts

```diff
@@ -0,0 +1,94 @@
+import axios from 'axios';
+import { expect } from 'chai';
+import { prisma } from '../infra';
+import deleteManyUsers from '../infra/utils/deleteUsers';
+
+describe('Creation mutation test', () => {
+  let userName = 'Jhon';
+  let userEmail = 'john@example.com';
+  let userPassword = 'password123';
+  let userBirthday = '00-00-000';
+
+  const query = `
+  mutation CreateUser($data: UserInput!) {
+    createUser(data: $data) {
+      name
+      email
+    }
+  }
+`;
+
+  const variables = {
+    data: {
+      name: userName,
+      email: userEmail,
+      password: userPassword,
+      birthDate: userBirthday,
+    },
+  };
+
+  it('should create a user mutation', async () => {
+    before(async () => {
+      await deleteManyUsers();
```

Comment:
Ao invés de deletar os usuários antes e depois de cada teste repetindo esse código você poderia adicionar ele num beforeEach e no afterEach fora antes do seus it.

Reply:
Boa, vou trocar.